### DISCUSSION TITLE
Landmark requirements for sample registration

### DISCUSSION THREAD

**Discussion by phiwei** [Borda/BIRL#43] [Q&A] [answered]
Hi, 

thank you for making this code publicly accessible. I am not sure if I understand the description correctly, I would like to try some of the registration methods that are included here with my own images. Do I need to generate landmarks or is this also possible without landmarks?

Thanks!

**Comment by Borda** [comment, marked as answer]
Hello, it may depend on what are your expectations from "to try some of the registration methods":
- to measure and see performance of particular methods on your data, then you need to have/make reliable landmark annotation
- just to see some image registration results, (I have not expected such usage of BIRL) then you can fake landmarks
   * creating landmark files without any point (right now I am not sure if there is consistency check)
   * passing some default landmark file for all image pairs

Does it answer your question? :)

**Comment by phiwei** [comment]
Hi, 

yes, thank you. Exactly, I don't want to measure performance for now. Ok, I will just create a file with some random points to work around that. Thanks again!
